control/attitude_error_estimator.cpp

```diff
@@ -51,8 +51,8 @@
  ******************************************************************************/
```

Comment:
I stop commenting on that, could you do it for all file name modification please ?